### PR TITLE
feat: [DAT-555] Add experimental feature "newCheckoutEnabled" for the new checkout

### PR DESCRIPTION
**Pull request by nreal**
# Background
We need to create a new feature flag "newCheckoutEnabled" to show the new checkout for the moment
[Documentation](https://github.com/FromDoppler/doppler-webapp/blob/01ce0ff67c8de962fe76cdf96a42c020c0e57b03/README.md#about-partial-functionalities)

We use the local storage to set the newCheckoutEnabled flag with the next command
`localStorage.setItem('dopplerExperimental', JSON.stringify({newCheckoutEnabled : true}));`

Pseudocode
newCheckout = true → show new checkout
newCheckout = false or undefined → show oldCheckout

### CODE REVIEW

**Comment by andresmoschini** [comment]
👍

**Comment by jhoffman-ms** [comment]
👍

**Review by bamaro-dev — COMMENTED** [review]
This is great!

**Comment by jcamposmk** [comment]
:+1:

File: src/components/Plans/PlanCalculator/PlanCalculatorButtons/index.js [hunk 1]
```diff
@@ -50,11 +60,20 @@ export const PlanCalculatorButtons = InjectAppServices(
...
+  controlPanelUrl,
+  planType,
+  planId,
+  discountId,
+  promoCode,
+  newCheckoutEnabled,
+}) => {
+  return newCheckoutEnabled
+    ? `/checkout/premium/${planType}?selected-plan=${planId}` +
+        `${discountId ? `&IdDiscountPlan=${discountId}` : ''}` +
```
Comment (jhoffman-ms) [inline src/components/Plans/PlanCalculator/PlanCalculatorButtons/index.js:73]: @nreal ,  Could you please rename the parameter: ```IdDiscountPlan``` by ```discountId```?. This parameter's name is expected in the checkout page.

Comment (nreal) [reply src/components/Plans/PlanCalculator/PlanCalculatorButtons/index.js]: Yes, it is ok done

**Pull request merged**